cmd/context/create.go

```diff
@@ -52,6 +54,21 @@ func NewContextCommand() *ContextCommand {
 		LoginController:      login.NewLoginController(),
 		OktetoClientProvider: okteto.NewOktetoClientProvider(),
 		OktetoContextWriter:  okteto.NewContextConfigWriter(),
+		IsOktetoKubeTokenPresent: func(oktetoURL string) (bool, error) {
```

Comment:
In a semantic way, I'd say yeah it makes a lot of sense. But seeing around the code dependencies, I feel that we'd be overloading the `OktetoInterface`. 

I think that we could refactor the code so that:
- `IsOktetoKubeTokenPresent` is in the `OktetoClient`
- [Provide](https://github.com/okteto/okteto/blob/1341b4675e0e546a3ce5d8ca86df54319933e83f/pkg/okteto/client.go#L62) returns an instance instead of an interface
- Reduce dependencies from the OktetoClient interface in other packages (optional, related)

Maybe we should do it in a follow up PR

WDYT?

Reply:
I like that approach

Reply:
I've created https://github.com/okteto/okteto/issues/3581to tackle this